addons/stock/models/stock_quant.py

```diff
@@ -569,13 +569,24 @@ def _get_quants_action(self, domain=None, extend=False):
         :param extend: If True, enables form, graph and pivot views. False by default.
         """
         self._quant_tasks()
+        # fixme: erase the following condition when it'll be possible to create a new record
+        # from a empty grouped editable list without go through the form view.
+        context = dict(self.env.context)
+        if not self.search_count([
+            ('company_id', '=', self.env.company.id),
+            ('location_id.usage', 'in', ['internal', 'transit'])
+        ]):
+            context.update({
+                'search_default_productgroup': 0,
+                'search_default_locationgroup': 0
+            })
```

Comment:
maybe do this only inventory mode?

Reply:
Indeed, it's useless to do that when the report isn't editable. Done :+1: